plugins/pnpm/README.md

```diff
@@ -0,0 +1,48 @@
+# pnpm plugin
+
+The pnpm plugin provides completion as well as adding many useful aliases.
+
+To use it, add pnpm to the plugins array of your zshrc file:
+
+```zsh
+plugins=(... pnpm)
+```
+
+## Aliases
+
+| Alias    | Command                      | Description                                                     |
+| :------- | :--------------------------- | :-------------------------------------------------------------- |
+| `pnpmg`  | `pnpm add -g`                | Install dependencies globally                                   |
+| `pnpmS`  | `pnpm add -S`                | Install and save to dependencies in your package.json           |
+| `pnpmD`  | `pnpm add -D`                | Install and save to dev-dependencies in your package.json       |
+| `pnpmF`  | `pnpm add -f`                | Force install from remote registries ignoring local cache       |
+| `pnpmE`  | `PATH="$(pnpm bin)":"$PATH"` | Run command from node_modules folder based on current directory |
+| `pnpmO`  | `pnpm outdated`              | Check which pnpm modules are outdated                           |
```

Comment:
Do we need capital acronyms? It could be tiring to control the shift key everytime.